target/linux/ramips/dts/W06.dts

```diff
@@ -0,0 +1,115 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "axing,w06", "mediatek,mt7628an-soc";
+	model = "Axing W06";
+
+	memory@0{
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wps {
+			label = "w06:green:wps";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+		};
+
+		wan {
+			label = "w06:green:wan";
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		};
+
+		wireless {
+			label = "w06:green:wlan";
+			gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&ehci {
+	status = "okay";
+};
+
+&ethernet {
+	mtd-mac-address = <&factory 0x28>;
+};
+
+&ohci {
+	status = "okay";
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		gpio {
+			ralink,group = "uart1", "wled_an", "p0led_an", "p1led_an", "p2led_an", "p3led_an", "p4led_an", "wdt", "refclk", "gpio", "i2s";
```

Comment:
Keep only `p2led_an`, `uart1` and  `wdt`. The GPIOs of the other groups are not used in your dts. The pinmux for the gpio group isn't required. This pin has only the GPIO function.

The fact that one uart1 GPIO (GPIO#46 [&gpio1 14]) instead if the wled_an GPIO (GPIO#44 [&gpio1 12]) is used look strange. Seem to me just another vendor oddity.

Reply:
I see, it is only the one that the device actually uses... I was misunderstanding.